config/routes.rb

```diff
@@ -2,17 +2,21 @@
   namespace :api do
     namespace :v1 do
       # Authentication
-      get '/auth/request', to: 'authentication#request_email_token'
-      get '/auth/verify', to: 'authentication#verify_email'
       post '/auth/sign_in', to: 'authentication#sign_in'
+      get '/auth/verify', to: 'authentication#verify_email'
+      get '/auth/request', to: 'authentication#request_email_token'
 
-      # Users - Admin
-      get '/users/all', to: 'users#index'
+      # Admin
+      get '/users/show', to: 'users#show_users'
+      get '/users/patients', to: 'users#show_patients'
       get '/users/show/:id', to: 'users#show_user'
+      delete '/users/destroy/:id', to: 'users#destroy_user'
 
-      # Users - Patients
-      get '/users/show', to: 'users#show_current_user'
-      post '/users/patients', to: 'users#create_patient'
+      # Patients
+      post '/users/patient', to: 'users#create_patient'
+      get '/users/patient/show', to: 'users#show_current_user'
+      delete '/users/patient/destroy', to: 'users#destroy_current_user'
+      get '/users/doctors', to: 'users#show_doctors'
```

Comment:
I am thinking you can merge this `patients` and `users`. 🤔

Reply:
Possibly, the reason why we used `/users/` here is for semantics and readability purposes only and to have a distinction between patients and doctors.

Reply:
okay understood po.